src/main/java/core/basesyntax/Report.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class Report {
+
+    private static final String NAME_SALARY_SEPARATOR = " - ";
+    private final String dataFrom;
+    private final String dataTo;
+    private final Map<String, Integer> workerNameSalaryMap;
+
+    public Report(String dataFrom, String dataTo, Map<String, Integer> workerNameSalaryMap) {
+        this.dataFrom = dataFrom;
+        this.dataTo = dataTo;
+        this.workerNameSalaryMap = workerNameSalaryMap;
+    }
+
+    @Override
+    public String toString() {
+        StringBuilder reportBuilder = new StringBuilder();
+        reportBuilder.append("Report for period ").append(dataFrom).append(" - ")
+                .append(dataTo).append('\n');
```

Comment:
Please, consider using `System.lineSeparator()` here instead of `'\n'`

[Code recommendations](https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info#use-systemlineseparator)

Reply:
Fixed. Screwed up with common mistakes, sry.